content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -17,22 +17,29 @@ In the following steps, we assume that the Istio components are installed and up
 1. [Download the new Istio release](/docs/setup/kubernetes/download-release/)
 and change directory to the new release directory.
 
-1. Upgrade Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-via `kubectl apply`, and wait a few seconds for the CRDs to be committed to the Kubernetes API server:
-
-{{< text bash >}}
-$ kubectl apply -f @install/kubernetes/helm/istio/templates/crds.yaml@
-{{< /text >}}
-
 ### Control plane upgrade
 
-The Istio control plane components include: Citadel, Ingress gateway, Egress gateway, Pilot, Policy, Telemetry and
+The Istio control plane components include: Citadel, Ingress gateway, Egress gateway, Pilot, Galley, Policy, Telemetry and
 Sidecar injector.
 
 #### Helm upgrade
 
 If you installed Istio with [Helm](/docs/setup/kubernetes/helm-install/#option-2-install-with-helm-and-tiller-via-helm-install) the preferred upgrade option is to let Helm take care of the upgrade:
 
+1. Upgrade the `istio-init` chart or install new release if it doesn't exists to keep all the Istio's [Custom Resource Definitions or CRDs for short](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) up-to-date:
```

Comment:
The text is difficult to understand because of an embedded conditional clause. I suggest splitting the sentence to make it easier to read. You find my suggestion below:
```suggestion
1. To keep updated all the Istio [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRDs), you must upgrade the `istio-init` chart. If the chart doesn't exist you must install the new release.
```
It would be better still if you could be specific or point to the release you mean the reader to install if the chart is not there.

Reply:
done